cmd/crank/beta/validate/cmd.go

```diff
@@ -0,0 +1,95 @@
+/*
+Copyright 2024 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package validate implements schema validation of Crossplane resources.
+package validate
+
+import (
+	"github.com/alecthomas/kong"
+
+	"github.com/crossplane/crossplane-runtime/pkg/errors"
+	"github.com/crossplane/crossplane-runtime/pkg/logging"
+)
+
+// Cmd arguments and flags for render subcommand.
+type Cmd struct {
+	// Arguments.
+	Schemas   string `arg:"" help:"A YAML file with XRD and CRD schemas."`
+	Resources string `arg:"" help:"A YAML file with Crossplane resources to validate."`
+
+	// Flags. Keep them in alphabetical order.
+	SkipSuccessLogs bool `help:"Skip printing success logs."`
+}
+
+// Help prints out the help for the render command.
+func (c *Cmd) Help() string {
+	return `
+This command validates Crossplane resources based on the schemas provided in offline mode.
```

Comment:
```suggestion
This command validates Crossplane resources based on the schemas provided.
```

I think the fact that this is "offline" is implied (and explained more clearly) below, where you say that it doesn't talk to crossplane.